lib/agent.js

```diff
@@ -37,7 +38,7 @@ var VMCache = require('./cache').VMCache;
 
 
 var VERSION = JSON.parse(fs.readFileSync(path.normalize(
```

Comment:
Is a comment needed w.r.t. why the use of split?  Also please make sure testing is in the ticket.

Reply:
Added a brief comment.